utilities/__test__/CojUtilities.test.ts

```diff
@@ -13,13 +17,22 @@ describe("CojUtilities", () => {
       );
     });
 
-    it("should return not signed when start date equal to coj epoch", () => {
-      expect(getStatusText("2023-08-01")).toEqual("Not signed");
+    it("should return Not signed, available from 02/05/2023 when start date equal to coj epoch and earlier than 13 weeks", () => {
+      const maxDate = new Date("2023-10-31");
```

Comment:
By my reckoning, this will fail on/after 2023-08-01 real time.
The "maxDate" test below is actually doing the same thing as this one without the risk of failure.